Model/src/BoardPackage/PropertySquare.java

```diff
@@ -1,7 +1,261 @@
+
+    /**An int to hold the cost of rent w/ two houses*/
+    private final int TWO_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ three houses*/
+    private final int THREE_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ four houses*/
+    private final int FOUR_HOUSE_RENT;
+
+    /**An int to hold the cost of rent w/ a hotel*/
+    private final int HOTEL_RENT;
+
+    /**An int to track how many houses are on the property*/
+    private int numHouses;
+
+    /**A boolean flag to track if the property is mortgaged*/
+    private boolean isMortgaged;
+
+    /**A Player variable to track what player owns this property*/
+    private Player owner;
+
+    /**An int to track the cost of purchasing one house for this property*/
+    private final int HOUSE_COST;
+
+    /**An int to track the cost of purchasing one hotel for this property*/
+    private final int HOTEL_COST;
+
+    /******************************************************************
+     * The constructor creates a Property and sets all the attributes
+     * related to the property.
+     * @param name the name of the PropertySquare
+     * @param price the PRICE of the property
+     * @param mortgageValue The mortgage value of the property.
+     * @param baseRent The rent cost of the property.
+     * @param oneHouseRent The rent cost of the property w/ one house.
+     * @param twoHouseRent The rent cost of the property w/ two houses.
+     * @param threeHouseRent The rent cost of the property w/ three houses.
+     * @param fourHouseRent The rent cost of the property w/ four houses.
+     * @param hotelRent The rent cost of the property w/ a hotel.
+     * @param numHouses The number of houses on the property.
+     * @param houseCost The cost of adding a house to the property.
+     * @param hotelCost The cost of adding a hotel to the property.
+     *****************************************************************/
+    public PropertySquare(String name, int price, int mortgageValue,
+                          int baseRent, int oneHouseRent,
+                          int twoHouseRent, int threeHouseRent,
+                          int fourHouseRent, int hotelRent,
+                          int numHouses, int houseCost,
+                          int hotelCost) {
+        super(name);
+
+        PRICE = price;
+        MORTGAGE_VAL = mortgageValue;
+
+        BASE_RENT = baseRent;
+        ONE_HOUSE_RENT = oneHouseRent;
+        TWO_HOUSE_RENT = twoHouseRent;
+        THREE_HOUSE_RENT = threeHouseRent;
+        FOUR_HOUSE_RENT = fourHouseRent;
+        HOTEL_RENT = hotelRent;
+
+        this.numHouses = numHouses;
+        isMortgaged = false;
+        owner = null;
+
+
+        HOUSE_COST = houseCost;
+        HOTEL_COST = hotelCost;
+    }
+
+    /******************************************************************
+     * The getPrice method returns the PRICE of this property.
+     *
+     * @return the PRICE of the property.
```

Comment:
Careful with JavaDoc comments. This will produce a comment that says return is "the" "PRICE of the property" it should be @return PRICE the price of the property. (@return <variable> <description>)